libgit2-sys/lib.rs

```diff
@@ -1100,6 +1116,7 @@ pub const GIT_DIFF_ENABLE_FAST_UNTRACKED_DIRS: git_diff_option_t = 1 << 14;
 pub const GIT_DIFF_UPDATE_INDEX: git_diff_option_t = 1 << 15;
 pub const GIT_DIFF_INCLUDE_UNREADABLE: git_diff_option_t = 1 << 16;
 pub const GIT_DIFF_INCLUDE_UNREADABLE_AS_UNTRACKED: git_diff_option_t = 1 << 17;
+pub const GIT_DIFF_INDENT_HEURISTIC: git_diff_option_t = 1 << 18;
```

Comment:
That was quite unexpected :) https://github.com/libgit2/libgit2/commit/2d9d246409d8131dfbbded2f880b784f205e93a5